ouroboros-consensus-test/src/Test/ThreadNet/Util/NodeTopology.hs

```diff
@@ -18,9 +20,12 @@ import           Data.Map.Strict (Map)
 import qualified Data.Map.Strict as Map
 import           Data.Set (Set)
 import qualified Data.Set as Set
+import           GHC.Generics (Generic)
 import           GHC.Stack (HasCallStack)
 import           Test.QuickCheck
 
+import           Quiet (Quiet (..))
```

Comment:
According to the style guide, you can group this import with the other imports above it :slightly_smiling_face:

Reply:
Fixed! Thanks.